app/assets/stylesheets/common/SyntaxHighlighter.scss

```diff
@@ -130,15 +130,15 @@
   }
 
   .footer {
-    background-color: #eceadb;
-    border-top: 1px solid #fff;
-    color: #333;
+    background-color: $background-support;
+    border-top: 1px solid $background-main;
```

Comment:
I'm not too sure where this is being used. The dp-about class this is nested within is only only used in shCore.js.

Reply:
You should change it. I'll investigate later, it might be that we can remove this whole `.dp-about` ruleset.